src/internal.rs

```diff
@@ -15,6 +15,25 @@
 //!
 //! When a participant is pinned, a `Guard` is returned as a witness that the participant is pinned.
 //! Guards are necessary for performing atomic operations, and for freeing/dropping locations.
+//!
+//! # Thread-local bag
+//!
+//! Objects that get unlinked from concurrent data structures must be stashed away until the global
```

Comment:
The following two paragraphs uses double spaces after a period, however the other text doesn't.